app/src/main/java/com/pajato/android/gamechat/exp/CheckersFragment.java

```diff
@@ -422,33 +426,43 @@ private ImageButton makeBoardButton(int index, int sideSize, Map<String, String>
         currentTile.setLayoutParams(param);
         String buttonTag = String.valueOf(index);
         currentTile.setTag(buttonTag);
+        float sp = sideSize / getResources().getDisplayMetrics().scaledDensity;
+        currentTile.setTextSize(COMPLEX_UNIT_SP, (float)(sp * 0.8));
+        currentTile.setTypeface(null, Typeface.BOLD);
+        currentTile.setGravity(Gravity.CENTER);
 
         // Handle the checkerboard positions.
```

Comment:
This comment and the next one can be combined without losing clarity.  A comment that covers 5-8 lines or so is optimal INHO.